testgrid/config/config.yaml

```diff
@@ -3201,6 +3226,10 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gce-device-plugin-gpu-stable1
     alert_options:
       alert_mail_to_addresses: 'gke-kubernetes-accelerators-bugs@google.com'
+  - name: gce-device-plugin-gpu-1-9
```

Comment:
All the tests should be listed in this `wg-resource-management` dashboard.

Reply:
done, thanks